src/content/en/tools/chrome-devtools/iterate/inspect-styles/css-animations.markdown

```diff
@@ -0,0 +1,177 @@
+---
+layout: shared/narrow
+title: "Inspect CSS animations"
```

Comment:
Can we use "Inspect animations"? CSS animations refer to `@keyframes rules` which can be confusing since CSS transitions & web animations are also supported. `requestAnimationFrame` are the only type of animations not supported.